package.json

```diff
@@ -5,7 +5,7 @@
   "main": "lib/index.js",
   "scripts": {
     "test": "ts-mocha --paths -p test/tsconfig.json \"test/**/*.ts\" --timeout 20000 --extension ts",
-    "build": "tsc"
+    "build": "concurrently \"tsc\" \"mkdir -p \"./lib\" && cp src/index.html lib/index.html\""
```

Comment:
does not have to be concurrent, could be `tsc; cp src/index.html lib/index.html`

Reply:
done